packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -257,6 +269,14 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
     if (event is PointerUpEvent || event is PointerCancelEvent)
       stopTrackingPointer(event.pointer);
   }
+
+  /// Whether or not a pointer is allowed to be tracked by this recognizer.
+  ///
+  /// Currently, it only checks for device kind. But in the future we could check
```

Comment:
This isn't really part of the customer documentation for this function, it's an implementation comment.

Move it to be inside the function body and make it a regular comment.